evap/evaluation/tests/tools.py

```diff
@@ -265,11 +265,12 @@ class LiveServerTest(SeleniumTestCase):
     selenium: WebDriver
     headless = True
     window_size = (1920, 3080)
-    serialized_rollback = True
+    serialized_rollback = False
```

Comment:
Why do we explicitly disable this again? Is there anything we should write down for future reference?

Reply:
django said that serialized rollbacks may slow down tests up to 3x. I tried that one out, but honestly I did not see a performance difference. The drawback of `serialized_rollback = False` is that we need to add the manager group ourselves (and maybe more stuff in the future).

I think we can even enable serialized rollbacks until we see a performance hit.

Reply:
I think that we either a) don't care about the setting, in which case it shouldn't be mentioned, or b) do care, in which case we should add a short comment why

Reply:
For reference: It seems like `serialized_rollback` is what I dug up from the docs: https://github.com/e-valuation/EvaP/pull/2077#discussion_r1486675190, although I didn't remember that either. Short comment in the code sure would help. Proposal:
> `TransactionTestCase` drops any data created by migrations, which we need for groups etc., so we need to fall back to `TestCase` and use `serialized_rollback`. See https://docs.djangoproject.com/en/5.0/topics/testing/overview/#test-case-serialized-rollback for details.

Reply:
I now added 
> SeleniumTestCase is a TransactionTestCase, which drops migration data. This keeps fixture data but may slow down tests, see https://docs.djangoproject.com/en/5.0/topics/testing/overview/#test-case-serialized-rollback